abs.yaml

```diff
@@ -0,0 +1,2 @@
+channels:
+  - https://staging.continuum.io/prefect/fs/pyproject-metadata-feedstock/pr3/df8cd6d
```

Comment:
abs.yaml can be removed as `pyproject-metadata 0.8.0` is already on the main channel

Reply:
done